inter-rater_variability/02a_rootlets_to_spinal_levels.py

```diff
@@ -1,24 +1,27 @@
 """
 The script does the following:
-    - project the nerve rootlets on the spinal cord segmentation to obtain spinal levels. This is done by dilating the
-    spinal cord segmentation by 1, 2 or 3 voxels (input argument) and then finding the intersection between the dilated
-    spinal cord segmentation and the rootlets segmentation. The spinal levels are then defined based on the top and
-    bottom slice of the intersection.
-    - compute the distance between the pontomedullary junction (PMJ) and the start and end of the spinal level (PMJ
-    label is required)
+    - project the nerve rootlets or vertebral labels on the spinal cord segmentation to obtain spinal levels
+    or vertebral levels. This is done by dilating the spinal cord segmentation by 1, 2 or 3 voxels (input argument)
+    and then finding the intersection between the dilated spinal cord segmentation and the rootlets segmentation.
+    The spinal levels are then defined based on the top and bottom slice of the intersection.
+    - compute the distance between the pontomedullary junction (PMJ) and the start and end of the spinal level
+    or vertebral level (PMJ label is required)
 
 The script outputs .nii.gz file with spinal levels and saves the results in CSV files.
 
 The script requires the SCT conda environment to be activated:
     source ${SCT_DIR}/python/etc/profile.d/conda.sh
     conda activate venv_sct
 
-Example:
-    python 02a_rootlets_to_spinal_levels.py -i sub-001_T2w_label-rootlet_rater1.nii.gz -s sub-001_T2w_seg.nii.gz -pmj sub-001_T2w_pmj.nii.gz
+Example for rootlets:
+    python 02a_rootlets_to_spinal_levels.py -i sub-001_T2w_label-rootlet.nii.gz -s sub-001_T2w_seg.nii.gz -pmj sub-001_T2w_pmj.nii.gz -type rootlets
+
+Example for vertebral labels:
+    python 02a_rootlets_to_spinal_levels.py -i sub-001_T2w_label-vertebral.nii.gz -s sub-001_T2w_seg.nii.gz -pmj sub-001_T2w_pmj.nii.gz -type vertebral
```

Comment:
Shouldn't it be `seg_labeled`?

```suggestion
    python 02a_rootlets_to_spinal_levels.py -i sub-001_T2w_seg_labeled.nii.gz -s sub-001_T2w_seg.nii.gz -pmj sub-001_T2w_pmj.nii.gz -type vertebral
```